### ISSUE TITLE
Improve text input accessibility on mobile

### ISSUE THREAD

**Issue by anyone00** [ajbura/cinny#773]
### Describe the bug

On mobile devices with narrow displays the space to enter text in too small.

see the screenshot below:
![Cinny's text input in room view](https://user-images.githubusercontent.com/41706133/185763887-2d5706b7-1646-4813-8f00-e03b73ad7caf.png)

### Reproduction

1. open cinny
2. click on a channel you can send a message in
3. the text input is too small for writing any text > 5 words

### Expected behavior

the text input should be at least make 2/3 words fit on a single line.

### Platform and versions

```shell
1. OS: Android 10
2. Browser: chrome 105, firefox 103
3. Cinny version: 2.1.2 (dev.cinny.in)
4. Matrix homeserver: matrix.org
```

### Additional context

I suggest:

1. reducing the spacing around the buttons
2. combine the emoji picker & stickers
3. put the sticker/emoji picker inside the text input
4. hide unnecessary icons when the text input is focused

here are some examples that can be inspired from:

<details>
<summary>minimal text input on mobile:</summary>

![Wide text input example](https://user-images.githubusercontent.com/41706133/185764119-64cb6b25-eb51-46a8-9287-dc12208e3055.png)

</details>

<details>
<summary>embed icons into the text input itself:</summary>

![wide text input example](https://user-images.githubusercontent.com/41706133/185764190-53dd9ad0-6352-4916-93a4-5a7f50f7c8db.png)

</details>

<details>
<summary>hide icons completely when text input is focused:</summary>

![example of Hidden icons when text input is focused](https://user-images.githubusercontent.com/41706133/185764240-f73b7b38-dbb8-46ec-9275-500d6d9fe9da.png)

</details>

<details>
<summary>hide icons completely when text input is focused:</summary>

![example 2 of Hidden icons when text input is focused](https://user-images.githubusercontent.com/41706133/185764266-0436780a-62eb-4c20-a77e-f31a127a3f3b.png)

</details>
